.github/workflows/publish-docs.yaml

```
@@ -60,7 +60,7 @@ jobs:
             m2r2 \
             make \
             myst-parser \
-            sphinx \
+            sphinx<8 \
```

```suggestion
            sphinx<8.0.0 \
```

I'd prefer it fully specified so conda doesn't try a bunch of wildcards